3741/3/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require_relative 'main_controller'
+
+class SessionsController < MainController
+  helpers do
```

Comment:
Declare helpers if you will use it in views, so if you are using them inside controller this just private methods

Reply:
i understand, replaced methods in private part